src/js/date-field.js

```diff
@@ -5,6 +5,21 @@ var parseDate = require('i18n-parse-date'),
 module.exports = require('ember-text-field').extend({
     autocomplete: 'off',
 
+    min: null,
+    max: null,
+
+    init: function() {
```

Comment:
init is already defined: https://github.com/billysbilling/ember-date-field/pull/3/files#diff-758f4c4248fb7fd991a7c5e3bf077f42R33

Do we really want to support dates s numbers?